app/src/main/java/app/gamenative/data/gog/GogRecommendationsRepository.kt

```
@@ -0,0 +1,363 @@
+                becausePlayed = because,
+                score = score,
+                seedCount = seedList.size,
+                seedIconUrl = seeds.values.firstOrNull { !it.isNullOrBlank() },
+                seedNames = seedList,
+            )
+        }
+    }
+
+    private fun formatCents(cents: Int): String {
+        if (cents <= 0) return "Free"
+        return CURRENCY_SYMBOL + String.format("%.2f", cents / 100.0)
```

detekt (1.23.8)

[warning] String.format("%.2f", cents / 100.0) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)